### PR TITLE
fix(invity): dark mode icons

### PR DESCRIPTION
**Pull request by vladimirvolek**
image
image

### CODE REVIEW

File: packages/suite/src/components/wallet/CoinmarketBuyProviderInfo/index.tsx [hunk 1]
```diff
@@ -9,10 +9,17 @@ const Wrapper = styled.div`
     align-items: center;
 `;
 
+const Bg = styled.div<{ color: string }>`
```
Comment (tomexx) [inline packages/suite/src/components/wallet/CoinmarketBuyProviderInfo/index.tsx]: not really important, maybe just my OCD but we use full-word naming of the tags everywhere :)

**Review by tomexx — APPROVED** [review]
just a naming nitpick

File: packages/suite/src/components/wallet/CoinmarketBuyProviderInfo/index.tsx (hunk 1 above)
Comment (mroz22) [inline packages/suite/src/components/wallet/CoinmarketBuyProviderInfo/index.tsx]: I think you don't need to pass props here. All styled components now have props.theme accessible by default. @slowbackspace shall correct me if I am wrong

**Review by mroz22 — CHANGES_REQUESTED** [review]

File: packages/suite/src/components/wallet/CoinmarketBuyProviderInfo/index.tsx (hunk 1 above)
Comment (vladimirvolek) [reply packages/suite/src/components/wallet/CoinmarketBuyProviderInfo/index.tsx]: yess, thanks!

**Review by mroz22 — APPROVED** [review]
looks good. does @slowbackspace have something to add?

**Review by slowbackspace — APPROVED** [review]

**Comment by slowbackspace** [comment]
lgtm

**Pull request merged**